org.OpenT2T.Sample.SuperPopular.Outlet/Wink Binary Switch/js/thingTranslator.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+var https = require('https');
+
+// logs device state
+function logDeviceState(device) {
+    if (typeof (device) !== 'undefined') {
+        console.log('  device.name          : ' + device.name);
+        console.log('  device.props         : ' + device.props);
+    } else {
+        console.log('device is undefined');
+    }
+};
+
+// Helper method to send power state commands to Wink
+function sendPowerStateCommandToDevice(powerState) {

```

Comment:
This looks like it is pretty common across a lot of Wink-type translators... no change for now, but I will set up time with you to pair-program a "helper" library that we can use to refactor these API calls into a single node module. We could fork this https://github.com/TheThingSystem/node-winkapi or start fresh if we need a different type of interface.